xcmds/ash/main.go

```diff
@@ -64,6 +65,11 @@ func main() {
 		log.Fatal(err)
 	}
 	r, err := t.Raw()
+	if err != nil {
+		// Error is non-fatal. The terminal might be missing some
+		// features, but can still be used.
+		log.Print(err)
```

Comment:
tell the user what this random error is about, e.g.

log.Printf("non-fatal terminal error: %v")

not sure if that's accurate